dev-system/src/cmr/dev_system/system.clj

```diff
@@ -100,13 +100,34 @@
 (def in-memory-elastic-log-level-atom
   (atom :info))
 
-(defmulti create-elastic
+(defmulti create-gran-elastic-server
   "Sets elastic configuration values and returns an instance of an Elasticsearch component to run
```

Comment:
I think the comment for this function and create-elastic-sever should make it clear what is contained in the cluster and what makes the two different. Right now they have the same comment